test/e2e/apimachinery/watchlist.go

```diff
@@ -148,20 +142,80 @@ var _ = SIGDescribe("API Streaming (aka. WatchList)", framework.WithSerial(), fe
 		streamedSecrets := secretList.Items
 		gomega.Expect(cmp.Equal(expectedSecrets, streamedSecrets)).To(gomega.BeTrueBecause("data received via watchlist must match the added data"))
 
+		ginkgo.By("Verifying if expected requests were sent to the server")
+		expectedRequestMadeByDynamicClient := getExpectedRequestMadeByClientFor(secretList.GetResourceVersion())
+		gomega.Expect(rt.actualRequests).To(gomega.Equal(expectedRequestMadeByDynamicClient))
+	})
+	ginkgo.It("doesn't support receiving resources as Tables", func(ctx context.Context) {
```

Comment:
does this test prove that requesting a table for streaming is not supported and that the fallback mechanism was triggered ?

Reply:
This assumes that list is using watchList underneath.
And later you rely on fallback to the regular list...

Verifying of 406 on the first request probably gives us what you want, but I think it all deserves a test-level comment to explain what/how is happening here.

Reply:
Ok, I will add a comment.

Reply:
done, ptal